client/src/components/Posts/Posts.js

```diff
@@ -45,6 +46,21 @@ const Posts = ({setCurrentId, setOpenCreatePost}) =>{
         }
     }
 
+    // function searchStringInArray (str, strArray) {
```

Comment:
If not needed can we remove it?

Reply:
Actually there were two types of search. But then it wasn't needed, so I commented.
Yes we can remove it.
I'll remove it and then push the code